src/pytest_plugins/consume/hive_simulators/conftest.py

```diff
@@ -143,7 +143,7 @@ def environment(
         blockchain_fixture.fork in ruleset
     ), f"fork '{blockchain_fixture.fork}' missing in hive ruleset"
     return {
-        "HIVE_CHAIN_ID": "1",
+        "HIVE_CHAIN_ID": blockchain_fixture.config.chain_id,
```

Comment:
```suggestion
        "HIVE_CHAIN_ID": str(Number(blockchain_fixture.config.chain_id)),
```

Need to make this cast here to make it work because all client mappers expect a decimal number as a string.